pre-commit-hooks.yaml

```
@@ -0,0 +1,16 @@
+# Define the hooks available in this repository
+- id: browserslist-lint
+  name: Validate Browserslist Configuration
+  # Command to execute the linter using npx for local installation
+  entry: npx browserslist-linter
+  # Specify the language for the hook; 'node' is used for npm-based tools
+  language: node
+  # Regular expression to match files that should trigger the hook
+  # The hook will run if `.browserslistrc` or `package.json` is modified
+  files: ^(\.browserslistrc|package\.json)$
```

```suggestion
  files: (\.browserslistrc|package\.json)$
```

some configs are not at the root